src/passwordHashing/generateHashPass.ts

```diff
@@ -0,0 +1,16 @@
+import * as  bcrypt from "bcrypt"
+import * as dotenv from "dotenv";
+
+//this script generate your hashed passord , change yourPassword , run the script then store -
+// the hashed value returned in the console in your .env file as PASSWORD
+dotenv.config();
+const yourPassword = "adminadmin"
+const hashPassword = async (yourPassword:string) => {
+    try {
+        const yourHashedPassword = await bcrypt.hash(yourPassword,10)
+        console.log({yourHashedPassword})
+    } catch (error) {
+        console.log('Errror when hashing the password', error)
```

Comment:
I see you have mixed of `"` and `'`, better to stick to one style, whichever you prefer!

Reply:
Jop